lib/html_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
```

Comment:
Connected to the above point, but slightly separate from it, what's your thinking behind adding this to the Sinatra namespace? It seems a little odd to me, though I suspect that's just because I'm missing something.